corehq/apps/domain/forms.py

```diff
@@ -2454,11 +2455,19 @@ def clean_version(self):
     def clean(self):
         app_id = self.cleaned_data.get('app_id')
         version = self.cleaned_data.get('version')
+        location_id = self.cleaned_data.get('location_id')
         if app_id and version:
             try:
                 self.version_build_id
             except BuildNotFoundException as e:
                 self.add_error('version', e)
+        enabled_release = get_latest_enabled_app_release(self.domain, location_id, app_id)
```

Comment:
check from here is later removed and only the model validation is reused, in the following commit.